### PR TITLE
feat: add room creation date in admin panel

### PR DESCRIPTION
**Pull request by Aseem22**
## Proposed changes (including videos or screenshots)

Rooms-Aseem (1)

## Further comments

https://rocketchat.atlassian.net/browse/CONN-272

### CODE REVIEW

File: apps/meteor/client/views/admin/rooms/RoomRow.tsx [hunk 1]
```diff
@@ -19,12 +20,13 @@ const roomTypeI18nMap = {
 const getRoomDisplayName = (room: Pick<IRoom, RoomAdminFieldsType>): string | undefined =>
 	room.t === 'd' ? room.usernames?.join(' x ') : roomCoordinator.getRoomName(room.t, room);
 
-const RoomRow = ({ room }: { room: Pick<IRoom, RoomAdminFieldsType> }) => {
+const RoomRow = ({ room }: { room: Pick<IRoom, RoomAdminFieldsType> &{ts ?: Date}}) => {
 	const t = useTranslation();
 	const mediaQuery = useMediaQuery('(min-width: 1024px)');
 	const router = useRouter();
+	const formatDate=useFormatDate();
```
Comment (hugocostadev) [inline apps/meteor/client/views/admin/rooms/RoomRow.tsx:27]: fix lint errors

File: apps/meteor/client/views/admin/rooms/RoomRow.tsx [hunk 2]
```diff
@@ -19,12 +20,13 @@ const roomTypeI18nMap = {
 const getRoomDisplayName = (room: Pick<IRoom, RoomAdminFieldsType>): string | undefined =>
 	room.t === 'd' ? room.usernames?.join(' x ') : roomCoordinator.getRoomName(room.t, room);
 
-const RoomRow = ({ room }: { room: Pick<IRoom, RoomAdminFieldsType> }) => {
+const RoomRow = ({ room }: { room: Pick<IRoom, RoomAdminFieldsType> &{ts ?: Date}}) => {
```
Comment (hugocostadev) [inline apps/meteor/client/views/admin/rooms/RoomRow.tsx:23]: add the `ts` prop to RoomAdminFieldsType type

**Review by hugocostadev — CHANGES_REQUESTED** [review]
check CI errors if any and fix the comments! Thanks for your contribution 🚀

File: apps/meteor/client/views/admin/rooms/RoomRow.tsx (hunk 2 above)
Comment (Aseem22) [reply apps/meteor/client/views/admin/rooms/RoomRow.tsx]: done

**Review by hugocostadev — CHANGES_REQUESTED** [review]
There are still CI issues, mostly Typescript problems

**Comment by hugocostadev** [comment]
Closing in favor of https://github.com/RocketChat/Rocket.Chat/pull/32709

**Pull request closed without merging**